addons/folo/common/src/main/java/org/commonjava/indy/folo/change/FoloTrackingListener.java

```diff
@@ -62,7 +63,7 @@
     private FoloConfig foloConfig;
 
     @Inject
-    private FoloRecordCache recordManager;
+    private FoloRecord recordManager;
```

Comment:
Well it is interface representing individual folo records. storage , manager is more implementation name.

Reply:
Well it should be domain name because. it is interface ( contract. ) defining general storage method which could. be. infinispan. or cassandra or postgre or....